# Use the same symbol verioning for hurd-i386 and kfreebsd-*
# Pushed upstream
--- libtheora-1.1.1/configure.ac.origin 2009-11-10 22:32:28.000000000 +0000
+++ libtheora-1.1.1/configure.ac    2009-11-10 22:38:03.000000000 +0000
@@ -222,7 +222,7 @@
     THC_VERSION_ARG="-export-symbols \$(top_srcdir)/win32/libtheora.def"
     AC_MSG_RESULT([-export-symbols])
     ;;
-  linux* | solaris* )
+  linux* | solaris* | gnu* | k*bsd*-gnu)
     THDEC_VERSION_ARG='-Wl,--version-script=$(srcdir)/Version_script-dec'
     THENC_VERSION_ARG='-Wl,--version-script=$(srcdir)/Version_script-enc'
     TH_VERSION_ARG='-Wl,--version-script=$(srcdir)/Version_script'
